staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -188,6 +192,15 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		delegate:  delegateHandler,
 	}
 	establishingController := establish.NewEstablishingController(s.Informers.Apiextensions().V1().CustomResourceDefinitions(), crdClient.ApiextensionsV1())
+	var sc storageversion.Client
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.StorageVersionAPI) &&
+		utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIServerIdentity) {
+		kubeclientset, err := kubernetes.NewForConfig(s.GenericAPIServer.LoopbackClientConfig)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create kubernetes clientset: %v", err)
```

Comment:
looks like the bots picked me from /test

should Kubernetes be uppercase here, or can it be dropped similar to line 173:
> return nil, fmt.Errorf("failed to create clientset: %v", err)

Reply:
Thanks for reviewing! `kubernetes` is the package name: `k8s.io/client-go/kubernetes.NewForConfig`. I have it to distinguish from L173, which is the apiextensions clientset: `k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset.NewForConfig`. Changed the error to `failed to create clientset for storage versions` to make it less confusing.